website/src/theme/DocPage/index.js

```diff
@@ -139,12 +139,12 @@ function DocPage(props) {
   );
 
     // DocPage Swizzle
-  useEffect(() => {
-      var instance1 = amplitude.getInstance().init(process.env.REACT_APP_AMPLITUDE_ID, null, {
-        apiEndpoint: `${window.location.hostname}/t`
-      })
-      amplitude.getInstance().logEvent('Docs Viewed', { "hostname": window.location.hostname, "path": location.pathname });
-  }, [location.pathname])
+  // useEffect(() => {
+  //     var instance1 = amplitude.getInstance().init(process.env.REACT_APP_AMPLITUDE_ID, null, {
+  //       apiEndpoint: `${window.location.hostname}/t`
+  //     })
+  //     amplitude.getInstance().logEvent('Docs Viewed', { "hostname": window.location.hostname, "path": location.pathname });
+  // }, [location.pathname])
```

Comment:
Should be in a separate PR since it's a fix, that could be reverted anytime

Reply:
There are multiple PRs in this one, that last commit was the biggest one by far.

I will run this by @slumbering and see how he wants to improve this specific part.

Thank you for the comments Tom! 🙌🏻

Reply:
Ok. What we need is a .env.local with an empty ﻿REACT_APP_AMPLITUDE_ID var.

Reply:
I create a PR for that

Reply:
PR #1941